### DISCUSSION TITLE
How to use a scene and background together?

### DISCUSSION THREAD

**Discussion by Pikku-a** [dialogic-godot/dialogic#2348] [Q&A] [answered]
I'm trying to add a background with an effect, so I made a scene for the effect. The scene loads, but there is no background.

Here is a video that shows this:
[dialogic-thing.webm](https://github.com/user-attachments/assets/dc88513f-2314-4f1d-b67a-802dde2f6bd8)

Here is an image of the timeline:
![Kuvakaappaus - 2024-07-19 12-49-05](https://github.com/user-attachments/assets/b2e1e0fb-afe0-42ef-b658-acb845d1981a)

It seems that I need to extend DialogicBackground in some way, but how?

**Comment by CakeVR** [comment, marked as answer]
Hello, yes, you need to extend the `DialogicBackground` class and have that as the root of your new scene.

You can learn more about this concept over here: https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#inheritance

**Reply by Pikku-a** [reply]
Thanks, I got it working.

In case someone else is having difficulties with this, I basically copied the default_background scene and edited it. That seemed to be the easiest way.
